### DISCUSSION TITLE
Codespaces down?

### DISCUSSION THREAD

**Discussion by mcknijak** [community/community#109756] [Codespaces]
### Select Topic Area

Bug

### Body

Hello,

Several students and I can't connect to Codespaces to work on Classroom Assignments. The following error always appears:

![image](https://github.com/community/community/assets/94488564/50b62562-0e9a-48df-a0c9-189e7e16d2fa)

Is there a Codespaces outage right now? If not, any advice on how to get the workbench to connect to the server?

**Comment by steveandkathyg** [comment]
I am getting exactly the same error. Started around 4:30 pm here in Southern California.

**Comment by jeanpaulcozzatti** [comment]
same.

**Comment by TheOtherTridentGuy** [comment]
I'm in Northern California and having the same issue: 

The workbench failed to connect to the server (Error: connection record disposed by ConnectionHub: {
"errorMessage": "Failed to start VS Code Remote server.",
"originalErrorType": "<no original type>",
"errorCode": -1,
"errorType": "GenericError",
"stack": "Error: Failed to start VS Code Remote server.\n at [CODESPACE_URL]\n at [CODESPACE_URL]\n at [CODESPACE_URL]\n at [CODESPACE_URL]\n at d ([CODESPACE_URL]\n at a ([CODESPACE_URL]"
} $1 $1)

**Reply by TheOtherTridentGuy** [reply]
I'm back up...
